src/main/java/picard/sam/FilterSamReads.java

```diff
@@ -332,19 +355,20 @@ protected int doWork() {
         checkInputs(Arrays.asList(Filter.includeReadList, Filter.excludeReadList), READ_LIST_FILE, "READ_LIST_FILE").ifPresent(errors::add);
         checkInputs(Collections.singletonList(Filter.includePairedIntervals), INTERVAL_LIST, "INTERVAL_LIST").ifPresent(errors::add);
         checkInputs(Collections.singletonList(Filter.includeJavascript), JAVASCRIPT_FILE, "JAVASCRIPT_FILE").ifPresent(errors::add);
+        checkInputs(Arrays.asList(Filter.includeTagValues, Filter.excludeTagValues), TAG, "TAG").ifPresent(errors::add);
```

Comment:
so passing in no values `TAG_VALUE` is technically valid.  You wont filter out any reads but it also won't fail because it seems the way the command line parser deals with `Array` input types is that by default it is an empty list as opposed to `null`.  These are the reasons I hadn't put it in as a validation check.  Would be curious if you feel it should still be checked (I'm open to either way)

Reply:
Yeah, I agree. though I'd emit a warning as it is almost certainly not what the user intended (similar to how DownsampleSam emits a warning when P=0 or P=1.)